draft-ietf-tls-dtls-connection-id.md

```diff
@@ -303,14 +305,45 @@ type
 version
 : This value contains the version number. 
 
+cid
+: Value of the negotiated CID.
+
+cid_length
+: 1 byte field indicating the length of the negotiated CID.
+
 length
 : This value contains the length information in the outer-header. 
 
+## CBC Block Ciphers
+
+This  specification updates the MAC calculation defined in 
+Section 6.2.3.2 of {{RFC5246}}, which is re-used by Section
+4.1.2.3 of {{RFC6347}}. 
+
+The MAC is generated as:
+
+   MAC(MAC_write_key, seq_num +
+                        TLSCompressed.type +
+                        TLSCompressed.version +
+						cid +
+						cid_length + 
+                        TLSCompressed.length +
+                        TLSCompressed.fragment);
```

Comment:
My feeling: for the terms used in this draft, `TLSCompressed` should be replaced by `DTLSCiphertext`, and the `cid` is a field of that.

(So, sorry for my irritating comment https://github.com/tlswg/dtls-conn-id/issues/46#issuecomment-468750119. My intention was just to show, that the definitions for the modes are very similar.)